build/webpack.dist.js

```diff
@@ -44,7 +44,6 @@ module.exports = [
       filename: '[name].js',
       chunkFilename: '[id].js'
     },
-    devtool: '#source-map',
```

Comment:
I thought devtool is using to debug and no need to enhance debug in production. But source map supports in production, reverted